src/lib/controllers/DevicesController.js

```diff
@@ -70,17 +102,17 @@ class DevicesController extends Controller {
         );
         return this.ok({ id: deviceID, status: 'Update started' });
       }
-console.log(postBody);
+
       // TODO not implemented yet
       // 3 flash device with precompiled binary
       if (postBody.file_type === 'binary' && this.request.files.file) {
-        this._deviceRepository.flashBinary(deviceID, this.request.files.file);
+        await this._deviceRepository.flashBinary(deviceID, this.request.files.file);
```

Comment:
I'm not sure yet, but as I know we should wait for `update started` status and then send ok response.